Sources/UtilityBeltUITesting/Models/AnimationConfiguration.swift

```diff
@@ -10,7 +10,12 @@ public struct AnimationConfiguration: LaunchEnvironmentCodable {
     /// Whether or not animations are enabled.
     public var animationsEnabled: Bool
     
-    public init(animationsEnabled: Bool) {
+    /// The value with which to set the application's key window layer speed.
+    /// See Apple's [documentation](https://developer.apple.com/documentation/quartzcore/camediatiming/1427647-speed) for more information.
```

Comment:
If you add a space here, it'll nicely put the "See more" stuff below the snippet when using Peek (or whatever the heck Apple calls it in Xcode) to view the docs on a method. Just a suggestion that I think improves documentation!

```suggestion
    /// The value with which to set the application's key window layer speed.
    ///
    /// See Apple's [documentation](https://developer.apple.com/documentation/quartzcore/camediatiming/1427647-speed) for more information.
```